src/main/viz/GenomeTrack.js

```diff
@@ -53,7 +53,7 @@ function renderGenome(ctx: DataCanvasRenderingContext2D,
         // We only push objects in the text case as it involves creating a
         // new object & can become a performance issue.
         // 0.5 = centered
-        ctx.fillText(letter, scale(1 + 0.5 + pos), height - 1);
+        ctx.fillText(letter, scale(1 + 0.5 + pos), height / 2);
```

Comment:
@snaheth , if you change the screen width it will cause text to render differently. I would recommend making the browser width larger and smaller and make sure that any changes accommodates these different widths.